PennMobile/Home/Fitness/Model/FitnessModels.swift

```diff
@@ -32,7 +32,9 @@ struct FitnessSchedule: Codable {
             self.name = try container.decodeIfPresent(FitnessFacilityName.self, forKey: .name) ?? .unknown
         } catch {
             self.name = .unknown
-            print("ERROR: Unknown fitness facility name")
+            if let unknownName = try? container.decodeIfPresent(String.self, forKey: .name) {
+                print("ERROR: Unknown fitness facility name - \(unknownName ?? "")")
+            }
```

Comment:
For some reason, the Fitness page is not loading schedules ... is that just on my end? If not, should we say `closed` or something? If its because their servers (or websites w/ the schedules) are down, should we consider removing the tab altogether for now?

Reply:
I vote for removing the tab altogether until we fix it. good call.

Reply:
removed tab